example/example.js

```diff
@@ -33,7 +33,7 @@ db.autoupdate('User', function(err) {
   });
```

Comment:
Need this line at the beginning of file
`var g = require('strong-globalize')();`